README.md

```diff
@@ -2,7 +2,7 @@
 Use Adyen's plugin for PrestaShop to offer frictionless payments online, in-app, and in-store.
 
 ## Integration
-The plugin integrates card component(Secured Fields) using Adyen Checkout for all card payments.
+The plugin integrates card component(Secured Fields) using Adyen Checkout for all card payments. Local/redirect payment methods are integrated with Checkout Payment Methods.
```

Comment:
let's call it local payment methods and remove the redirect. As this is how we group all payment methods now.

Reply:
```suggestion
The plugin integrates card component(Secured Fields) using Adyen Checkout for all card payments. Localpayment methods are integrated with Checkout Payment Methods.
```